ethereum/p2p/src/test/resources/keys/tls_context_factory/README.md

```diff
@@ -0,0 +1,214 @@
+Following instructions were used to set up various certificates and keystores.
+
+The certificates hierarchy 
+~~~
+                 root.ca.besu.com
+                        | 
+                 inter.ca.besu.com
+                        |      
+compa.ca.besu.com                compb.ca.besu.com
+       |                               |
+avocado.compa.besu.com        banana.compb.besu.com
+apricot.compa.besu.com        blueberry.compb.besu.com 
+~~~
+
+CompanyA clients
+Client Avocado is valid. (backed by PKCS11/nssdb as well)
```

Comment:
IMO no need for the extra indirection of the fruit names, ClientA is more readable.

Reply:
now that I look back into it, it does sound silly ... will revert back to using partner1client1, parner1client2 naming scheme.